packages/clima_ui/lib/screens/weather_screen.dart

```diff
@@ -142,46 +161,56 @@ class WeatherScreen extends HookConsumerWidget {
                   }
                 }(),
               ),
-              child: Container(
-                constraints: const BoxConstraints.expand(),
-                child: SingleChildScrollView(
-                  physics: const BouncingScrollPhysics(),
-                  child: Column(
-                    children: [
-                      const MainInfoWidget(),
-                      Divider(
-                        color: Theme.of(context)
-                            .textTheme
-                            .subtitle1!
-                            .color!
-                            .withAlpha(65),
-                      ),
-                      SizedBox(
-                        height: MediaQuery.of(context).size.aspectRatio == 1.0
-                            ? 24.h
-                            : 16.h,
-                        child: const HourlyForecastsWidget(),
-                      ),
-                      Divider(
-                        color: Theme.of(context)
-                            .textTheme
-                            .subtitle1!
-                            .color!
-                            .withAlpha(65),
-                      ),
-                      const DailyForecastsWidget(),
-                      Divider(
-                        color: Theme.of(context)
-                            .textTheme
-                            .subtitle1!
-                            .color!
-                            .withAlpha(65),
-                      ),
-                      const AdditionalInfoWidget(),
-                    ],
-                  ),
-                ),
-              ),
+              child: fullWeatherState is w.Error &&
+                      fullWeatherState.fullWeather == null
+                  ? FailureBanner(
+                      failure: fullWeatherState.failure,
+                      onRetry: fullWeatherStateNotifier.loadFullWeather,
+                    )
+                  : fullWeatherState.fullWeather == null
+                      ? null
+                      : Container(
```

Comment:
Not related to this PR, but why is this `Container` present?

Reply:
I don't remember why, but we can decide on it later I guess.